vispy/visuals/volume.py

```diff
@@ -348,7 +357,9 @@
                 color = $sample(u_volumetex, iloc);
                 if (color.g > u_threshold) {
                     color = calculateColor(color, iloc, dstep);
-                    gl_FragColor = $cmap(color.g);
+                    val = max(color.g - clim.x, 0);
```

Comment:
wasn't entirely sure if this is the best place to implement clims for the ISO rendering... thoughts welcome